zhaquirks/nodon/__init__.py

```diff
@@ -1 +1,60 @@
 """NODON module for custom device handlers."""
+
+from zigpy.quirks import CustomCluster
+import zigpy.types as t
+from zigpy.zcl.foundation import (
+    BaseAttributeDefs,
+    BaseCommandDefs,
+    DataTypeId,
+    Direction,
+    ZCLAttributeDef,
+    ZCLCommandDef,
+)
+
+
+class NodOnPilotWireMode(t.enum8):
+    """Pilot wire mode."""
+
+    # Codes taken from
+    # https://github.com/Koenkk/zigbee-herdsman-converters/blob/0f4833340a20db3dae625a61c41d9be0a6f952be/src/converters/fromZigbee.ts#L5285.
+
+    Off = 0x00
+    Comfort = 0x01
+    Eco = 0x02
+    FrostProtection = 0x03
+    ComfortMinus1 = 0x04
+    ComfortMinus2 = 0x05
+
+
+NODON = "NodOn"
+NODON_PILOT_WIRE_CLUSTER_ID = 0xFC00  # 64512
+
+
+class NodOnPilotWireCluster(CustomCluster):
```

Comment:
All of this seems specific to the pilot wire device, so I'd probably just move it into the `pilot_wire.py` file (above the `QuirkBuilder`) and leave this file empty (except for the docstring at the top).

Reply:
@TheJulianJES, I have prepared a re-newed PR with everything moved to pilot_wire.py. I just didn't have time to push it. I've pushed it now.